packages/icons/src/FlairIcon.tsx

```diff
@@ -20,9 +20,19 @@ const FlairIcon = ({
   gradient = 'purpleToBlue',
   ...props
 }: FlairIconProps) => {
+  const getIconSize = () => {
+    let iconSize: IconProps['size'] = children.props.size;
+
+    if (iconSize === 'xlarge' || iconSize === 'huge' || !iconSize) {
```

Comment:
Curios why we want to restrict any sizes in general?

Reply:
xlarge and huge go outside the confines of the flair -- I can make it so the FlairIcon is responsive to the inner icon size, would that be preferable?

Reply:
If it isn't too much of a pain I'd say so. Would be nice to have equivalence with the standard `Icon` component.